Examples/src/main/java/com/mapbox/vision/examples/ArActivity.java

```diff
@@ -344,4 +347,30 @@ public void userOffRoute(Location location) {
     private List<Point> buildStepPointsFromGeometry(String geometry) {
         return PolylineUtils.decode(geometry, Constants.PRECISION_6);
     }
+
+    private static final Map<String, ManeuverType> maneuversMap = new HashMap<>();
+
+    static {
+        maneuversMap.put("turn", ManeuverType.Turn);
+        maneuversMap.put("depart", ManeuverType.Depart);
+        maneuversMap.put("arrive", ManeuverType.Arrive);
+        maneuversMap.put("merge", ManeuverType.Merge);
+        maneuversMap.put("on ramp", ManeuverType.OnRamp);
+        maneuversMap.put("off ramp", ManeuverType.OffRamp);
+        maneuversMap.put("fork", ManeuverType.Fork);
+        maneuversMap.put("roundabout", ManeuverType.Roundabout);
+        maneuversMap.put("exit roundabout", ManeuverType.RoundaboutExit);
+        maneuversMap.put("end of road", ManeuverType.EndOfRoad);
+        maneuversMap.put("new name", ManeuverType.NewName);
+        maneuversMap.put("continue", ManeuverType.Continue);
+        maneuversMap.put("rotary", ManeuverType.Rotary);
+        maneuversMap.put("roundabout turn", ManeuverType.RoundaboutTurn);
+        maneuversMap.put("notification", ManeuverType.Notification);
+        maneuversMap.put("exit rotary", ManeuverType.RotaryExit);
+    }
+
+    private ManeuverType mapToManeuverType(@Nullable final String maneuver) {
```

Comment:
Could you just use `switch` with strings here? Similarly to `when` in kotlin part

Reply:
@alexander-kulikovskii advised to use map. IMHO it also looks a bit better.

Reply:
Well, if you'll format `switch` similarly to how is `when` formatted, using single line for case-return (`case "turn": return ManeuverType.Turn;`) it will be short && look nice. And you won't need separate function with logic (and possibility to make mistake, too)